models/articles.models.js

```diff
@@ -23,7 +46,7 @@ const fetchArticles = () => {
   FROM articles
   LEFT JOIN comments ON comments.article_id = articles.article_id
   GROUP BY articles.article_id
-  ORDER BY articles.created_at DESC`
+  ORDER BY ${sort_by} ${order}`
```

Comment:
Really nice job with this model, really clean! Defaulting the parameters means you don't have to keep if/else'ing around the sort_by and order.